src/plugin.ts

```diff
@@ -24,6 +44,16 @@ const plugin: FastifyPluginAsync = async (fastify) => {
     },
```

Comment:
I would assume just one process (bad words) for this PR, it's already quite complex. I'll deal with multiple processes in future PR. I think to choose which processes to run, currently, it's probably only hard-coded. If we want to configure the processes, we need some interface. (It would be nice to have it in admin mode, but we don't have it up-to-date so far).